WorldMoodTracker/js/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import WorldMoodTracker from './components/WorldMoodTracker.js';
 
-export const host = 'http://loklak.org';
+export const host = 'http://35.192.111.68';
```

Comment:
Keep a check as soon as this feature is introduced in the loklak server. Please change it.